samples/apps-in-meeting/csharp/README.md

```diff
@@ -0,0 +1,54 @@
+# ContentBubbleSample
+This sample illustrates how to implement Content Bubble In-Meeting Experience.
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+2) If you are using Visual Studio
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `apps-in-meeting/csharp` folder
+  - Navigate to `ContentBubbleBot` folder
+  - Select `ContentBubbleBot.csproj` file
+
+3) Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http -host-header=rewrite 3978
+    ```
+### Create a bot with Azure Bot Service 
+1. This bot has been created using [Bot Framework](https://dev.botframework.com).
+2. Navigate to app registration page, in the overview section, copy and save the **Application (client) ID**. You’ll need those later when updating your Teams application manifest and in the appsettings.json.
+3. Under **Manage**, select **Expose an API**. 
+4. Select the **Set** link to generate the Application ID URI in the form of `api://{AppID}`. Insert your fully qualified domain name (with a forward slash "/" appended to the end) between the double forward slashes and the GUID. The entire ID should have the form of: `api://fully-qualified-domain-name/{AppID}`
+    * ex: `api://%ngrokDomain%.ngrok.io/00000000-0000-0000-0000-000000000000`.
+5. Navigate to **API Permissions**, and make sure to add the follow permissions:
+-   Select Add a permission
+-   Select Microsoft Graph -\> Delegated permissions.
+    * User.Read (enabled by default)
+    * email
+    * offline_access
```

Comment:
These look like steps for SSO. Could you please add relevant steps for this sample?

Reply:
Removed this added specific steps for configuring content bubble bot